drivers/storage/portworx/status.go

```diff
@@ -109,7 +109,11 @@ func (p *portworx) updateStorageNodes(
 	// If cluster is running internal kvdb, get current bootstrap nodes
 	kvdbNodeMap := make(map[string]*kvdb_api.BootstrapEntry)
 	if cluster.Spec.Kvdb != nil && cluster.Spec.Kvdb.Internal {
-		strippedClusterName := strings.ToLower(configMapNameRegex.ReplaceAllString(cluster.Name, ""))
+		clusterID := cluster.Name
+		if cluster.Annotations != nil && cluster.Annotations[pxutil.AnnotationClusterID] != "" {
+			clusterID = cluster.Annotations[pxutil.AnnotationClusterID]
+		}
```

Comment:
Can we just extract this in a util function like this - `pxutil.GetClusterName(cluster)`?